paasta_tools/cli/cmds/spark_run.py

```diff
@@ -458,9 +488,12 @@ def get_spark_env(
     return spark_env
 
 
-def _parse_user_spark_args(spark_args: Optional[str]) -> Dict[str, str]:
+def _parse_user_spark_args(
+    spark_args: Optional[str], enable_k8s_autogen: bool = True
```

Comment:
should this default to the same value in https://github.com/Yelp/paasta/pull/3210/files#diff-30f8e9c33224158fab1e77efc4004f7d2344230ea7b4ffbf64b3fca21513cc00R142

Reply:
Yep, it doesn't actually impact the running codepath and was only defaulted for tests, but I'll switch it to false 👍